package.json

```diff
@@ -12,36 +12,42 @@
     "build:css": "bin/build-css",
     "build": "bin/build",
     "lint": "standard",
-    "unit-test": "lab -v -c -r console -o stdout -r lcov -o coverage/lcov.info",
+    "unit-test": "bin/run-unit-tests",
+    "pretest": "npm run docker:start-test-double-infrastructure",
     "test": "npm run lint & npm run unit-test",
-    "postinstall": "npm run build"
+    "posttest": "npm run docker:stop-test-double-infrastructure",
+    "postinstall": "npm run build",
+    "docker:start-infrastructure": "docker compose -f docker/infrastructure.yml up -d",
+    "docker:start-test-double-infrastructure": "docker compose -f docker/infrastructure.yml up -d fws-app-redis",
+    "docker:stop-infrastructure": "docker compose -f docker/infrastructure.yml down",
+    "docker:stop-test-double-infrastructure": "docker compose -f docker/infrastructure.yml rm -f -s fws-app-redis"
   },
   "author": "",
   "license": "ISC",
   "dependencies": {
-    "@hapi/bell": "12.3.0",
-    "@hapi/boom": "9.1.4",
-    "@hapi/catbox-redis": "6.0.2",
-    "@hapi/cookie": "11.0.2",
+    "@hapi/bell": "13.1.0",
+    "@hapi/boom": "10.0.1",
+    "@hapi/catbox-redis": "7.0.2",
+    "@hapi/cookie": "12.0.1",
     "@hapi/good": "9.0.1",
     "@hapi/good-console": "9.0.1",
     "@hapi/good-squeeze": "6.0.0",
-    "@hapi/hapi": "20.2.1",
-    "@hapi/inert": "6.0.5",
+    "@hapi/hapi": "21.4.2",
+    "@hapi/inert": "7.1.0",
     "@hapi/joi": "17.1.1",
```

Comment:
https://www.npmjs.com/package/@hapi/joi @hapi/joi has been deprecated btw, joi is the current package.